### DISCUSSION TITLE
需求和建议

### DISCUSSION THREAD

**Discussion by fzf404** [fzf404/Monit#46] [Ideas]
欢迎大家来这里提出需求和建议！

**Comment by fzf404** [comment]
功能建议：电脑录屏，截屏、服务器监控（在线状态、核心温度、连接延迟）、Spotify 音乐播放、倒计时、课程表。
修改建议：图像插件（放大缩小、拖放、旋转）、插件设置（缩放、透明度、颜色、字体）、MacOS 图标。

**Comment by Theodore-wgh** [comment]
功能建议：能不能做一个跟fence软件功能比较接近的，文件夹可视化映射到，以及桌面分区的功能
